icicle/tests/test_hash_api.cpp

```diff
@@ -628,15 +913,25 @@ TEST_F(HashApiTest, MerkleTreeLeavesOnDeviceTreeOnHost)
   }
 }
 
+void deep_copy_byte_array_to_vec(const std::byte* src, std::size_t size, std::vector<std::byte>& dest)
```

Comment:
There is already a simpler way to do that : 
'auto dest = std::vector<std::byte>(src, src + size);'

Reply:
Didn't know - will do!